index.html

```diff
@@ -13,16 +13,34 @@
 
     <script>
         var calculator = {
-            addNumbers: function () {
+            addNumbers: function (numberOne, numberTwo) {
                 // hint: these are strings
                 var numberOne = document.forms.inputs.numberOne.value;
                 var numberTwo = document.forms.inputs.numberTwo.value;
-                document.forms.inputs.result.value = numberOne + numberTwo;
+                document.forms.inputs.result.value = parseInt(numberOne) + parseInt(numberTwo);
             },
-            subtractNumbers: function () {
+            subtractNumbers: function (numberOne, numberTwo) 
```

Comment:
The formatting makes this code look like it needs to be reformatted.
Tabbing, spacing, new lines, indentation, etc are all key to readable code.

You can use the built-in formatting in VS Code.

To use the formatter, go to the View > Command Palette:

<img width="579" alt="Screen Shot 2020-06-16 at 10 23 49 PM" src="https://user-images.githubusercontent.com/718718/84847982-474ea580-b020-11ea-99c1-f87c67d480f8.png">

Then choose the "Format Document" option.

<img width="322" alt="Screen Shot 2020-06-16 at 10 24 00 PM" src="https://user-images.githubusercontent.com/718718/84848000-4f0e4a00-b020-11ea-987a-c06eba3717cb.png">

This should make the code have an easier to read formatting. Push that code up as a new commit.